### ISSUE TITLE
Touch not working on  Firefox/Chrome for Windows 8 tablet

### ISSUE THREAD

**Issue by a-shrub** [react-spring/react-use-gesture#136]
Touch gestures don't work on anything past 7.0.0-beta.7 on Windows 8 tablet for latest Firefox and Chrome. Version 7.0.0-beta.8 will run briefly and die with an InvalidPointerId (tested by incriminating dependencies on https://codesandbox.io/s/react-use-gesture-simple-i5e0j until it broke).

**Comment by dbismut** [comment]
Hey @a-shrub thanks for raising the issue and taking the time to check which version breaks. It’s going to be very hard for me to debug since I’m not equipped with this hardware so can you please let me know if there’s any specific log in the console?

Also are these tablets with both touch and mouse inputs?

**Comment by dbismut** [comment]
Hey @a-shrub I've released `7.0.4` that adds `touch` and `mouse` listeners. I had removed it at one point because it seems like iOS 13.1 beta would respond to both touch and mouse listeners, resulting in events being firing twice (therefore harming performance). It looks like it was a beta bug that they fixed for the final release as I tested now and it looks like everything works fine.

Let me know if this fixes it for you.
